packages/@tailwindcss-upgrade/src/migrate.ts

```diff
@@ -36,12 +37,13 @@ export async function migrateContents(
 
   return postcss()
     .use(migrateAtApply(options))
-    .use(migrateThemeToVar(options))
     .use(migrateMediaScreen(options))
     .use(migrateAtLayerUtilities(stylesheet))
     .use(migrateMissingLayers())
     .use(migrateTailwindDirectives(options))
     .use(migrateConfig(stylesheet, options))
+    .use(migrateBorderCompatibility(options))
+    .use(migrateThemeToVar(options))
```

Comment:
Using `theme(…)` inside of the compatibility CSS, so needed to move this after the `migrateBorderCompatibility` migration.